RELEASE/scripts/autoscend/auto_consume.ash

```diff
@@ -1429,6 +1434,78 @@ boolean auto_knapsackAutoConsume(string type, boolean simulate)
 	return true;
 }
 
+boolean auto_chewAdventures()
+{
+	//tries to chew a size 4 familiar spleen item for adventures
+
+	//these items are all out of standard except in Quantum Terrarium
+	if(!is_unrestricted($item[Unconscious Collective Dream Jar]) && !is_unrestricted($item[Powdered Gold]) && !is_unrestricted($item[beastly paste]))
+	{
+		//powdered gold is also restricted in G lover, but if all these are restricted this must be standard
+		//no point continuing if standard restrictions apply
+		return false;
+	}
+
+	//called to eat size 4 spleen items when adventures < 10 and we can't eat or drink anything
+
+	int oldSpleenUse = my_spleen_use();
+
+	//do we wait until the last adventure to chew spleen?
+	boolean waitUntilLastAdventure = false;
+	if(item_amount($item[stench jelly]) >= 2)
+	{
+		int spleenForStenchJelly = min(my_adventures(),item_amount($item[stench jelly]));
+		//wait if chewing 4 spleen would take away the chance to use 2 or more stench jellies
+		if (oldSpleenUse - spleenForStenchJelly <= 2)
+		{
+			waitUntilLastAdventure = true;
+		}
+	}
+	if(auto_havePillKeeper() && spleen_left() == 6)
```

Comment:
This condition is only true if we have the pillkeeper and exactly 6 spleen left. This should be checking if we have the pillkeeper and 6 or more spleen left otherwise it could consume stuff at 7-15 spleen remaining using up spleen which could be better utilised for pillkeeper charges.

Reply:
So apparently this is on purpose for a case where you have exactly 6. 
if you had 7 then you can chew 1 paste and 1 pill keeper. or use 2 pill keeper
using 1 paste costs you 1 pillkeeper

if you had 5 you could can chew 1 paste and 1 pillkeeper, or use 2 pill keeper
using 1 paste costs you 1 pillkeeper

if you have 6 you can chew 1 past or use 2 pill keeper
using 1 paste costs you 2 pillkeeper

Moreover, all it does is force us to wait until just 1 adventure is left (above the currently set reserve limit) rather than chew immediately as we drop to 9 adv. just in case we happen to want to use a pill keeper to force a noncombat in those 7 adv remaining before reaching 1+auto_advToReserve()

hmmm... thinking about it, adv giving spleen chewables are sufficiently meh that I think we could simplify the code to just always wait until we are exactly at 1+auto_advToReserve() to consume one of them.

Reply:
> hmmm... thinking about it, adv giving spleen chewables are sufficiently meh that I think we could simplify the code to just always wait until we are exactly at 1+auto_advToReserve() to consume one of them.
> @Malibu-Stacey what do you think?

I agree. The size 4 spleen items are 1.875 adv/spleen which is garbage tier for food and booze. If we have absolutely no adventures left and we have spleen items to consume, consuming them is better than not (as any user with half a brain would manually do that and then re-run the script as I have done countless times in G-Lover & QT) but consuming them before that point isn't going to make a difference and could be detrimental to PillKeeper or future spleen using IotM usage (assuming the dev's ever bother with spleen again).

Reply:
change made.